### ISSUE TITLE
Functionality to export a report of the data once calculated

### ISSUE THREAD

**Issue by ru4en** [SasView/sasview#2109]
**Is your feature request related to a problem?**
A quick and easy way to export readable data such as calculations, graphs and images to a location selected by the user could be helpful as the current methods involves a lot of right clicking, coping and pasting. The other option involves saving the project as a json file which is hard to read. Just wanted to note that this functionality has been prevouly requested  in #1362 and #1992. I beleve that @lucas-wilkins has already started to work on the importing/exporting parameters aspect of this. 

**Solution**
A button to export data from the prospective panels could be implemented which get the data from the output box located withing the prospective panels and appending it with the existing Data info functionality. Additionally, a ```ctrl+e``` shortcut could also be implemented to improve user experience. 
**Describe alternatives you've considered**
Maybe creating a class for exploring data for the different types of analysis (eg: Corfunc, fitting, invariant, Inversion) could be useful as that would make any further additions to the output easier to implement down the road. 

**Additional context**
I will upload an example of a report and Ui design for reffrence

**Comment by lucas-wilkins** [comment]
Please see me about this, I've done a lot in this direction in a not-yet merged branch. See #1992

**Comment by lucas-wilkins** [comment]
I thought some more about this since we spoke, #2065 doesn't get merged soon, you could just create the branch you'll work in from the 1992... working branch i.e. click "create a branch" in the "Development" section on the right ->, and then select the issue 1992 branch as the branch source , like this:

![image](https://user-images.githubusercontent.com/9784906/175783150-4bb0f49f-acd3-4fa8-92fc-1154f9af0a9d.png)

**Comment by ru4en** [comment]
yes I might do that this week after working on the batch data loading for Pr.

**Comment by ajj** [comment]
Should this be closed now that #2065 has been merged and instead have issues for each perspective to now implement reports?

**Comment by dehoni** [comment]
Showing a summary of the results and saving reports is implemented for batches.
